agent/newrelic-install.sh

```diff
@@ -1038,9 +1038,19 @@ for this copy of PHP. We apologize for the inconvenience.
 
   case "${pi_ver}" in
      7.0.*)
+      warning_message="${pdir}: Support for PHP '${pi_ver}' in the New Relic PHP agent is deprecated."
```

Comment:
Are the single quotes required around pi_ver?

Reply:
They're not required - it's purely a matter of visual preference. The only reason the quotes are there, is because that's the format of the message that was used when PHPs 5.5 and 5.6 were deprecated. Here's the comparison of the message with the quotes:
```
/usr/local/bin: Support for PHP '7.0.33' in the New Relic PHP agent is deprecated.
```
and without the quotes:
```
/usr/local/bin: Support for PHP 7.0.33 in the New Relic PHP agent is deprecated.
```

Reply:
Personally, I think it looks better without, but it's fine either way.

Reply:
I'd vote for removing the single quotes.